src/main/resources/view/DarkTheme.css

```
@@ -95,7 +94,7 @@
   -fx-font-family: "Monaco", "Menlo", "Consolas", "Courier New", monospace;
   -fx-font-size: 13px;
   -fx-text-fill: #c9d1d9;
-  -fx-padding: 8 16 8 16;
+  -fx-padding: 20 16 12 16;
```

Let's change this to
```suggestion
  -fx-padding: 24 16 12 16;
```

This way, the top and bottom paddings look the same